src/game/common/const.mjs

```diff
@@ -122,6 +122,26 @@ export const GameTreasure = {
   rsg: { type: 'rsg', label: 'RSG', tile: 6, value: 200, baseVal: 1, denomination: 0 },
 };
 
+export const TEAMS = {
+  cyber_timberloards: { id: 0, name: 'Cyber Timberloards', color: '#7af8ff' },
+  neon_lumberjacks: { id: 1, name: 'Neon Lumberjacks', color: '	#d875ff' },
+  mechanical_builders: { id: 2, name: 'Mechanical Builders', color: '#8d4be0' },
+  code_crackers: { id: 3, name: 'Code Crackers', color: '#39ff14' },
+  glitch_hackers: { id: 4, name: 'Glitch Hackers', color: '#1f51ff' },
+};
+
+export const TEAM_BONUS = 5000;
+
+export const DEFAULT_TEAMS_CONFIG = {
+  on: false,
```

Comment:
is this flag needed? doesn't amount = 0 mean "teams off"?
right now someone can set inconsistent values (like on: true, amount 0 or on: false, amount: 5)